pkg/registry/apps/statefulset/storage/storage_test.go

```diff
@@ -42,6 +43,7 @@ func newStorage(t *testing.T) (*REST, *StatusREST, *etcdtesting.EtcdTestServer)
 // createStatefulSet is a helper function that returns a StatefulSet with the updated resource version.
 func createStatefulSet(storage *REST, ps apps.StatefulSet, t *testing.T) (apps.StatefulSet, error) {
 	ctx := genericapirequest.WithNamespace(genericapirequest.NewContext(), ps.Namespace)
+	ctx = genericapirequest.WithRequestInfo(ctx, testhelper.FakeRequestInfo("", ""))
```

Comment:
that's the behaviour now in `qualifiedResourceFromContext` (it wasn't before). @shiywang does that mean we can remove some of the `WithRequestInfo` calls?